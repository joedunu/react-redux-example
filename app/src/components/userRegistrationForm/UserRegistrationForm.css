.userRegistrationForm {
    background-color: white;
    padding: 40px;
    & input {
        padding: 10px;
        margin-left: 10px;
    }
}