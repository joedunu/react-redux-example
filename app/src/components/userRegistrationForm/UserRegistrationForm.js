import React from 'react'
import styles from './UserRegistrationForm.css'

const UserRegistrationForm = () => (
  <div className={styles.userRegistrationForm}>
        First Name: <input />
        Last Name: <input />
        Date of Birth: <input />
  </div>
)

export default UserRegistrationForm
