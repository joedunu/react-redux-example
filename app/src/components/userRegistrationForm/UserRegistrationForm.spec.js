import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'
import UserRegistrationForm from './UserRegistrationForm'

test('Search snapshot test', () => {
  const component = shallow(<UserRegistrationForm />)
  const tree = shallowToJson(component)
  expect(tree).toMatchSnapshot()
})
