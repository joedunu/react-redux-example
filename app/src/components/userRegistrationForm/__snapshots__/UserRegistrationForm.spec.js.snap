// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search snapshot test 1`] = `
<div>
  First Name: 
  <input />
  Last Name: 
  <input />
  Date of Birth: 
  <input />
</div>
`;
